src/add-field-dropdown/add-field-dropdown.component.ts

```diff
@@ -48,6 +49,10 @@ export class AddFieldDropdownComponent implements OnChanges {
     private domUtilService: DomUtilService,
     private keysStoreService: KeysStoreService) { }
 
+  get schemaProperties() {
+    return _.pickBy(this.schema.properties, value => !value.hidden);
```

Comment:
what would be better is to remove the `differentKeys` pipe from the template, then do
```typescript
this.addableKeys = Set.fromKeys(schema.properties)
  .substract(this.fields)
  .filter(field => !schema.properties[field].hidden) 
```

on `ngOnChanges`